include/dali/c_api.h

```diff
@@ -309,6 +309,33 @@ DLL_PUBLIC void daliDeletePipeline(daliPipelineHandle *pipe_handle);
  */
 DLL_PUBLIC void daliLoadLibrary(const char *lib_path);
 
+  /**
+   * @brief Returns the epoch size with/without padded samples for given, named reader
```

Comment:
```suggestion
   * @brief Returns the epoch size with/without padded samples for given named reader
```
I think the comma is out of place here...

Reply:
Done